functions/http/http-form-data/src/test/java/functions/HttpFormDataTest.java

```diff
@@ -113,10 +108,9 @@ public void functionsHttpFormData_shouldSaveFiles() throws IOException {
   @Test
   public void functionsHttpFormData_shouldProcessFields() throws IOException {
     when(request.getMethod()).thenReturn("POST");
-    when(request.getParts()).thenReturn(new HashMap<>());
+    when(request.getParts()).thenReturn(Map.of());
```

Comment:
@lesv @kurtisvg @saturnism is there a `checkstyle` config that checks for "outdated" syntax like this?

Reply:
Not as far as I know.